source/ceylon/time/timezone/parser/iana/parseMonth.ceylon

```diff
@@ -0,0 +1,11 @@
+import ceylon.time.base {
+    months,
+    Month
+}
+
+shared Month parseMonth(String month) {
+    "Invalid Month for parse in timeZone"
+    assert(exists currentMonth = months.all.find((Month elem) 
+        => elem.string.lowercased.startsWith(month.trimmed.lowercased)));
```

Comment:
Huum, but I only saw 3 chars months in  all files, I dont believe they have 2 chars because this way its really impossible to know which month they want.